### PR TITLE
Update and add full contract code

### PR DESCRIPTION
**Pull request by vivekjain23**
This PR consists of changes in the EVM How-To docs, adding full contract code to all articles that previously did not have it, maintaining consistency in Contract Code Documentation in the EVM How-To Section.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [X] I have performed a self-review of my own changes
- [X] I have made sure that added/changed links still work
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by lucas-tortora — APPROVED** [review]

**Comment by lucas-tortora** [comment]
@Ginowine and @salaheldinsoliman , could you please review this?

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/erc20-native-token.md [hunk 1]
```diff
@@ -35,11 +35,20 @@ NativeTokenID memory id =token.nativeTokenID();
...
+
+import "@iota/iscmagic/ISC.sol";
+
+contract NativeToken {
+
+    function nativeTokenID(uint32 _foundrySN) public view returns (bytes memory) {
+        ERC20NativeTokens token = ERC20NativeTokens(
+            ISC.sandbox.erc20NativeTokensAddress(_foundrySN)
+        );
+        ISC.NativeTokenID memory id = token.nativeTokenID();
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/erc20-native-token.md:50]: Why did that line change from `NativeTokenID memory id =token.nativeTokenID();`
to ` ISC.NativeTokenID memory id = token.nativeTokenID();` ?

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/mint-token.md [hunk 2]
```diff
@@ -29,13 +29,25 @@ ISC.accounts.mintNativeTokens(_foundrySN, _amount, allowance);
...
+
+import "@iota/iscmagic/ISC.sol";
+
+contract NativeTokenMinter {
+    event MintedNativeTokens(uint32 foundrySN, uint amount);
+
+    function mintNativeTokens(uint32 _foundrySN, uint _amount, uint64 _storageDeposit) public payable {
+        require(msg.value == _storageDeposit * (10 ** 12), "Please send exact funds to pay for storage deposit");
+        
+        ISC.ISCAssets memory allowance;
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/mint-token.md:44]: Same in this line

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md [hunk 3]
```diff
@@ -43,14 +43,31 @@ address erc20address = ISC.sandbox.erc20NativeTokensAddress(_foundrySN);
...
+        string calldata _symbol,
+        uint8 _decimals,
+        uint64 _storageDeposit
+    ) public payable {
+        require(msg.value == _storageDeposit * (10 ** 12), "Please send exact funds to pay for storage deposit");
+
+        ISCAssets memory allowance;
+        allowance.baseTokens = _storageDeposit;
+
+        ISC.registerERC20NativeToken(_foundrySN, _name, _symbol, _decimals, allowance);
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md:68]: `ISC.sandbox.registerERC20NativeToken(_foundrySN, _name, _symbol, _decimals, allowance);`

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md [hunk 4]
```diff
@@ -43,14 +43,31 @@ address erc20address = ISC.sandbox.erc20NativeTokensAddress(_foundrySN);
...
+        address erc20address = ISC.erc20NativeTokensAddress(_foundrySN);
```
Comment (salaheldinsoliman) [inline docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md:69]: `address erc20address = ISC.sandbox.erc20NativeTokensAddress(_foundrySN);`

**Review by salaheldinsoliman — CHANGES_REQUESTED** [review]
@vivekjain23  What IDE are you using to compile the contracts?

File: docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md (hunk 4 above)
Comment (vivekjain23) [reply docs/build/isc/v1.1/docs/how-tos/core-contracts/token/register-token.md:69]: My bad, I intended to fix this in the next commit but forgot to push the changes.

**Review by salaheldinsoliman — APPROVED** [review]
@vivekjain23 LGTM

**Pull request merged**